### ISSUE TITLE
[WCAG 2.1]A11y_Java Tooling for IntelliJEclipse_Customize New Project_Color Contrast: Non text contrast for file icon is 1.7:1 which is <3:1

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3321]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.11; #Non-text Contrast; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.4.11

**NOTE: Same issue is observed in all the screens where file icon is present**

**Repro Steps: ​**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ' in the Marketplace and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and click new-->Project
9) In the New Project dialog box, select Maven and press enter
10) Check "Create from archetype" check box and select "maven-archetype-webapp" and tab till Next and press enter
11) Specify the GroupId and ArtifactId and press Next button
12) Tab till "Override" checkbox after "User Setting Files" text-box and press space to check it
13) Check the color contrast ratio for the File icon in the enabled text-box

**Actual Result:**
Non text contrast for file icon is 1.7:1 which is <3:1

**Expected Result:**
Non text contrast for file icon should be >=3:1

**User Impact:**
Users with visual impairments will find it difficult to access the icon

**MAS Reference:**
[MAS 1.4.11](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=12fdff15-141a-4609-91f3-af19fc946d83)
![Bug_3_IntelliJ_Customize_Non Text Contrast](https://user-images.githubusercontent.com/41108985/62204195-5ac23500-b3aa-11e9-98c5-a34bdbc76208.png)

**Comment by Flanker32** [comment]
The background color of JTextField are affected by IntelliJ theme, which is designed by IntelliJ and out of the scope of Toolkits. Besides, IntelliJ provide three theme by default and one of them is High contrast, which is designed for low vision users by high color contrast ratio, here is the screen shots.

![image](https://user-images.githubusercontent.com/12445236/63936056-83187e80-ca91-11e9-844c-c813e5e88fdd.png)

![image](https://user-images.githubusercontent.com/12445236/63936106-9deaf300-ca91-11e9-8408-4b93a5a57d46.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium, low vision users may can not find the icon to browse file.
- Quality of experience: not impacted. IntelliJ provide alter theme for users including the High contrast one with high color contrast ratio.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@soumyatippapuram Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
Reported a bug to IntelliJ IDEA (https://youtrack.jetbrains.com/issue/IDEA-222109)

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749756
